### PR TITLE
Some cleanups for ubuntu hosts

### PR DESCRIPTION
**Pull request by GheRivero**

### CODE REVIEW

**Comment by runcom** [comment]
Awesome, thanks a lot! I'll test this out and merge it!

File: cri-o.yml [hunk 1]
```diff
@@ -54,8 +54,11 @@
         - pkgconfig
         - json-glib-devel
         - skopeo-containers
-    - name: update on ubuntu
-      command: "apt-get update"
```
Comment (runcom) [inline cri-o.yml:58]: this is breaking with:
```
 _____________________________
< TASK [Update all in Ubuntu] >
 -----------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

skipping: [192.168.122.57]
skipping: [192.168.122.227]
fatal: [192.168.122.205]: FAILED! => {"changed": false, "failed": true, "msg": "Could not find aptitude. Please ensure it is installed."}
```

Comment (GheRivero) [reply cri-o.yml:58]: Fixed

Comment (runcom) [reply cri-o.yml:58]: gonna re-test it, thanks a lot!

**Comment by runcom** [comment]
works, thanks again :)

**Pull request merged**